code/dqc/pctAvgDailyChange.q

```diff
@@ -10,7 +10,7 @@
 pctAvgDailyChange:{[fname;tabname;rt;ndays;thres]                                       
   .lg.o[`pctAvgDailyChange;"Checking daily average change"];  
   if[ndays>-1+count .Q.pv; :(0b;"error: number of days exceeds number of available dates")];
-  previous:select avg resvalue from rt where date in (-1+last date;(-1*ndays)+last date),funct=fname,table=tabname;
+  previous:select avg resvalue from rt where date within((-1*ndays)+last date;-1+last date),funct=fname,table=tabname;
```

Comment:
think this would work too:

```suggestion
  previous:select avg resvalue from rt where date within(-1*ndays;-1)+last date,funct=fname,table=tabname;
```